shell/agents/AIShell.Azure.Agent/AzPS/AzPSChatService.cs

```diff
@@ -9,7 +9,7 @@ namespace AIShell.Azure.PowerShell;
 
 internal class AzPSChatService : IDisposable
 {
-    internal const string Endpoint = "https://azclitools-copilot.azure-api.net/azps/api/azure-powershell/copilot/streaming";
+    internal const string Endpoint = "https://azclitools-copilot-apim-temp.azure-api.net/azps/copilot/streaming";
```

Comment:
The az-ps endpoint returns "out of scope" for legit questions with this endpoint. Is it expected?

![image](https://github.com/user-attachments/assets/81512825-8549-4817-bc6f-1427c0aebcfb)

Reply:
Thanks! It's not expected. I just update the service's prompt to fix it.